### DISCUSSION TITLE
Sponsor profile stuck waiting for approval

### DISCUSSION THREAD

**Discussion by wtdcode** [community/community#110464] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My sponsor application has been waiting for approval for more than a few weeks. Any reason?

Location: Singapore

**Comment by uditmahato** [comment]
To address the issue of your sponsor profile being stuck waiting for approval on GitHub, there are a few steps you can take. Firstly, reach out directly to GitHub support through their help center or support channels, providing details about your sponsor profile and the prolonged approval delay. Additionally, log in to your GitHub account and check for any status indicators or messages related to sponsorship applications to see if there's any specific reason for the delay. Review the documentation or guidelines provided by GitHub regarding the sponsorship approval process to ensure all requirements have been met and accurate information has been provided in your application. Lastly, exercise patience but don't hesitate to follow up with GitHub support if the delay seems unusually long, as they can provide insights into the status of your application and expedite the approval process if necessary.

**Comment by wtdcode** [comment, marked as answer]
Solved by opening a GitHub ticket.
